<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>missionAspectDialog</class>
 <widget class="QDialog" name="missionAspectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>174</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>236</width>
    <height>174</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>236</width>
    <height>174</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Arc aspect</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelArcName">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditArcName">
       <property name="text">
        <string>loitering</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelColor">
       <property name="text">
        <string>Color: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxColorPicker">
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorYELLOW.png</normaloff>:/icons/colorYELLOW.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorRED.png</normaloff>:/icons/colorRED.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorGREEN.png</normaloff>:/icons/colorGREEN.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorBLUE.png</normaloff>:/icons/colorBLUE.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorMAGENTA.png</normaloff>:/icons/colorMAGENTA.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorWHITE.png</normaloff>:/icons/colorWHITE.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorCYAN.png</normaloff>:/icons/colorCYAN.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorGRAY.png</normaloff>:/icons/colorGRAY.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark Red</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorDARKRED.png</normaloff>:/icons/colorDARKRED.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Dark Yellow</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorDARKYELLOW.png</normaloff>:/icons/colorDARKYELLOW.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark Blue</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorDARKBLUE.png</normaloff>:/icons/colorDARKBLUE.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark Magenta</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorDARKMAGENTA.png</normaloff>:/icons/colorDARKMAGENTA.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Light Gray</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorLIGHTGRAY.png</normaloff>:/icons/colorLIGHTGRAY.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark Gray</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorDARKGRAY.png</normaloff>:/icons/colorDARKGRAY.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark Cyan</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorDARKCYAN.png</normaloff>:/icons/colorDARKCYAN.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorBLACK.png</normaloff>:/icons/colorBLACK.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hide</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/colorTRANSPARENT.png</normaloff>:/icons/colorTRANSPARENT.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelModel">
       <property name="text">
        <string>Model:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxModel">
       <item>
        <property name="text">
         <string>Default</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-DEFAULT.png</normaloff>:/icons/STAModelsIcons-DEFAULT.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XMM</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-XMM.png</normaloff>:/icons/STAModelsIcons-XMM.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISS</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-ISS.png</normaloff>:/icons/STAModelsIcons-ISS.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rosetta</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-ROSETTA.png</normaloff>:/icons/STAModelsIcons-ROSETTA.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Planck</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-PLANCK.png</normaloff>:/icons/STAModelsIcons-PLANCK.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Soyuz</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-SOYUZ.png</normaloff>:/icons/STAModelsIcons-SOYUZ.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ExoMars</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-EXOMARS.png</normaloff>:/icons/STAModelsIcons-EXOMARS.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ariane 5</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-ARIANE5.png</normaloff>:/icons/STAModelsIcons-ARIANE5.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Station</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-GROUNDSTATION.png</normaloff>:/icons/STAModelsIcons-GROUNDSTATION.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Point</string>
        </property>
        <property name="icon">
         <iconset resource="../../iconary/sta-icons.qrc">
          <normaloff>:/icons/STAModelsIcons-POINT.png</normaloff>:/icons/STAModelsIcons-POINT.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBoxMissionAspect">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../iconary/sta-icons.qrc"/>
  <include location="../../iconary/sta-icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBoxMissionAspect</sender>
   <signal>accepted()</signal>
   <receiver>missionAspectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxMissionAspect</sender>
   <signal>rejected()</signal>
   <receiver>missionAspectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
